Mathlib/CategoryTheory/Triangulated/Opposite/Basic.lean

```diff
@@ -99,9 +99,6 @@ lemma shiftFunctorAdd'_op_hom_app (X : Cᵒᵖ) (a₁ a₂ a₃ : ℤ) (h : a₁
   erw [@pullbackShiftFunctorAdd'_hom_app (OppositeShift C ℤ) _ _ _ _ _ _ _ X
     a₁ a₂ a₃ h b₁ b₂ b₃ (by dsimp; omega) (by dsimp; omega) (by dsimp; omega)]
   rw [oppositeShiftFunctorAdd'_hom_app]
-  obtain rfl : b₁ = -a₁ := by omega
-  obtain rfl : b₂ = -a₂ := by omega
-  obtain rfl : b₃ = -a₃ := by omega
```

Comment:
This changes seems fine; the tauto change also (assuming it benchmarks).

Reply:
Ack